sky/packages/sky/lib/src/material/material_app.dart

```diff
@@ -28,6 +28,8 @@ class MaterialApp extends StatefulComponent {
     this.onGenerateRoute
   }) : super(key: key) {
     assert(routes != null);
+    assert(routes.containsKey('/'));
+    assert(!routes.values.any((RouteBuilder builder) => builder(new RouteArguments.dummy()) == null));
```

Comment:
I removed this assert and guarded the invocation of the builder with an assert and a console message reporting the failed route.